### DISCUSSION TITLE
Way to change commands

### DISCUSSION THREAD

**Discussion by TimTamCoder** [drizzle-mizzle/CharacterAI-Discord-Bot#148] [Ideas]
I don't know much about discord bots, I've made like 1 and thats it. is there a way I can change commands or can one be added? specifically just being able to say [clear] to reset the bot and make it forget your conversation.

**Comment by drizzle-mizzle** [comment]
Nothing stops you from modifying it in any way you want. All nedeed info about text commands can be found here:
https://discordnet.dev/guides/text_commands/intro.html

**Reply by drizzle-mizzle** [reply]
But I would suggest you to switch on the new bot I'm working on in the current moment:
https://github.com/drizzle-mizzle/Character-Engine-Discord

**Reply by TimTamCoder** [reply]
I was going to take a look at that one, is there any releases or does the source code function the same
edit: nevermind
